modules/examples/flake/flake.nix

```diff
@@ -16,9 +16,8 @@
         [ pkgs.vim
         ];
 
-      # Auto upgrade nix package and the daemon service.
-      services.nix-daemon.enable = true;
-      # nix.package = pkgs.nix;
+      # Uncomment this if you'd like to use a different version of Nix.
+      # nix.package = pkgs.nixVersions.nix_2_18;
```

Comment:
Nix 2.18 is about to go out of upstream’s support policy at the end of the month so I’m a bit worried about this. Maybe we could just drop the line after all, or change it to `nixVersions.git` so it won’t go stale.

Reply:
Removed it 👍